p4-16/spec/P4-16-spec.mdk

```diff
@@ -3958,8 +3961,14 @@ assignments, in comparisons, in field selection expressions, and as
 arguments to functions, method or actions.  Structure-valued
 expressions are not left values.
 
-The compiler must raise an error if a field name appears more than
-once in the same structure-valued expression.
+Structure-valued expressions must provide a value for every member of
+the struct or header type to which it evaluates, either by doing so
+explicitly with a literal expression of the appropriate type, or by
```

Comment:
this could be slightly clearer

Reply:
@mbudiu-vmw @jnfoster I have replaced this unclear paragraph with 3 shorter paragraphs that are hopefully clearer.